### ISSUE TITLE
Module '@11ty/eleventy' has no exported member 'EleventyRenderPlugin'

### ISSUE THREAD

**Issue by anantakrishna** [11ty/eleventy#2935]
### Operating system

Windows 10

### Eleventy

2.0.1

### Describe the bug

Requiring any of the plugins bundled with the core code triggers the TS2305 error in [VSCode with the `@ts-check`](https://code.visualstudio.com/docs/nodejs/working-with-javascript#_type-checking-javascript) directive:

For example, https://www.11ty.dev/docs/plugins/render/#installation.

```js
const { EleventyRenderPlugin } = require('@11ty/eleventy');
```

> `Module '"@11ty/eleventy"' has no exported member 'EleventyRenderPlugin'.ts(2305)`

Probably this happens because Eleventy's `index.d.ts` contains only `UserConfig`:
```ts
import UserConfig from "./UserConfig";

export { UserConfig };
```

This syntax seems to be a workaround:

```js
const EleventyRenderPlugin = require('@11ty/eleventy/src/Plugins/RenderPlugin.js');
```

### Reproduction steps

1. Open the project in VSCode.
2. Install Eleventy into project.
3. Add `EleventyRenderPlugin` into the `.eleventy.js` config file according to https://www.11ty.dev/docs/plugins/render/#installation.
4. Add `//@ts-check` directive on top of the `.elevent.js` file
5. See the TS2305 error on `EleventyRenderPlugin`.

### Expected behavior

It is expected that TypeScript language server (VSCode) would see all the bundled plugins as exported from the `@11ty/eleventy` package.

### Screenshots

![image](https://github.com/11ty/eleventy/assets/6065071/0ada227d-ace4-4fbc-8c64-56a3e7fd4392)

**Comment by zachleat** [comment]
#3060 has been merged, milestoned to 3.0

**Comment by mayank99** [comment]
Looks like this was "un-fixed" by #3291. I no longer have types for `import { EleventyRenderPlugin } from "@11ty/eleventy"`

**Comment by zachleat** [comment]
For future folks this was fixed by #3349
